docs/book/src/testing/testing-with-rust.md

```diff
@@ -52,6 +52,9 @@ placeholder in the template. Otherwise, `cargo-generate` automatically converts
 With `--force`, this means that both `my_fuel_project` and `my-fuel-project` are valid project names,
 depending on your needs.
 
+> _**Note**: `templates/sway-test-rs` can be replaced with `templates/sway-script-test-rs` or `templates/sway-predicate-test-rs` to generate a test
```

Comment:
Feels like this is becoming a CLI

```
$ forc test generate
Please choose a template:
- contract
- script
- predicate
```

cc @sdankel WDYT

Reply:
I am weary of introducing a TUI like this that we need to officially maintain. I would prefer that `forc` stay language agnostic for the time being. 

I can see the desire for wanting something like this when giving workshops using the rust sdk. Forc allows 3rd parties to write custom plugins. Perhaps this could be a community plugin.

Reply:
Yeah semantically it also makes sense to reserve this only for cargo generate. If someone wants to use rust alongside with sway, they will have cargo install and thus can generate a project for it. If I don't want to use rust for my tests forc should not generate template with rust code in it. As sway and rust are different languages, using sway should not necessarily mean that you need to use rust as well

Reply:
I agree. `forc test` should be reserved for tests written in sway.